_posts/2019-10-10-event-store-packages-for-amazon-linux-2.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: blog-post
+title: Event Store packages for Amazon Linux 2
+author: Arwin Neil Baichoo
+---
+# Event Store packages for Amazon Linux 2
+
+An Event Store Commercial package for Amazon Linux 2 is now available!
+
+#### Changes moving from Amazon Linux 1 to Amazon Linux 2 Packages
+---
+
+#### Phasing out support for AWS Auto Scaling groups 
+Running Event Store in AWS EC2 Auto Scaling groups (AGS) is no longer a recommended practice because AGS can end up simultaneously or near simultaneously deleting all instances

```

Comment:
```suggestion
Running Event Store in AWS EC2 Auto Scaling groups (AGS) is no longer a recommended practice because ASG can end up simultaneously or near simultaneously deleting all instances
```